Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -175,80 +253,63 @@ _Note_: the `executable` could in principle also be given via a URL; however, th
-
-```yaml
-name: SCS Open IaaS
-url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Certification/scs-open-iaas.yaml
-prerequisite:
-  name: SCS Compatible IaaS
-  url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Tests/scs-compatible-iaas.yaml
-variables:
-  - os_cloud
-versions:
-  - version: v5 # This version is in a draft state and work in progress
-    # No stabilized_at: date set yet
-    standards:
-      - name: Flavor naming
-        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0100-v2-flavor-naming.md
-        condition: mandatory # is default and can be left out
-        checks:
-          - executable: flavor-name-check.py
-            env:
-              OS_CLOUD: "{os_cloud}"
-            id: flavor-name-check
-            lifetime: day
-      - name: Image metadata
-        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0102-v1-image-metadata.md
-        condition: mandatory
-        checks:
-          - executable: image-md-check.py
-            args: -c {os_cloud} -v
-            id: image-md-check
-            lifetime: day
-          - executable: image-md-check2.py
-            condition: optional
-            id: image-md-check-2
-            lifetime: day
-      - name: Standard images
-        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0104-v1-standard-images.md
-        parameters:
-          image_spec: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Tests/iaas/scs-0104-v1-images.yaml
-        checks:
-          - executable: ./iaas/standard-images/images-openstack.py
-            args: -c {os_cloud} -d {image_spec}
-            id: standard-images-check
-  - version: v4 # This is the upcoming version with a given target date. No further changes should be done to this set of standards
-    stabilized_at: 2022-04-01
-    standards:
-      - name: ....
-
-  - version: v3 # This is the stable set of standards that is currently active
-    stabilized_at: 2021-10-01
-    deprecated_at: 2022-11-08
-    standards:
-      - name: ....
-
-  - version: v2 # This set of standards is obsolete and has been replaced by v3
-    stabilized_at: 2021-07-01
-    deprecated_at: 2021-11-01
-    standards:
-      - name: ....
-```
+### Test-case descriptor
+
+The following fields are valid for every check descriptor:
+
+| Key               | Type            | Description                                                                                                       | Example           |
+| ----------------- | --------------- | ----------------------------------------------------------------------------------------------------------------- | ----------------- |
+| `id`              | String          | Identifier for this test case (immutable and unique within this module)                                           | `image-md-check`  |
```

Comment:
Proposal: use `ref` or something when things with an `id` are *referenced*, and only use `id` key when something with a document wide unique id is defined. Makes it easier to grep through and search for things, and makes use vs. definition distinction obvious as well.

Reply:
68e60de75d37bfd060d0fcc5e0115b47e85db576